app/views/posts/index.html.erb

```diff
@@ -1,2 +1,70 @@
-<h1>List All Posts of a User</h1>
-<p>All posts will appear here</p>
+<h1>List Recent Posts of a User</h1>
+<%# <p>All posts will appear here</p> %>
+
+<main id="posts" class="page">
+<div id="user-page">
+ <div class="btn-container">
+      <%= link_to "Home", root_path %>
+
+
+      <div class="back-link-container">
+        <%= link_to "Back", :back, class: "back-link2" %> 
+    </div> 
+    </div>
+</div>
+
+<div class="user-container posts">
+<div class="image-div">
+ <%= image_tag('user.png', :width => "100px", :heigth=>"100px") %>
+ </div>
+ <div class="userdetails-div">
+<h3><%= @user.name %></h3>
+<p class="posts-counter">Number of posts: <%= @user.posts_counter%></p>
+</div>
+</div>
+<%# End of user profile %>
+
+ <article class="post-article">
+<% @posts.each do |post| %>
+<div class="post-container">
+<h3 class="post-title"><%= link_to post.title, "posts/#{post.id}" %></h3>
+<div class="post-details">
+              <p class="details"><%= post.text[0..100]... %></p>
+            </div>
+ <div class="comment-like-container">
+              <p class="coment-likes-data">
+                Comment: <%= post.comments_counter || 0 %>, Likes: <%= post.likes_counter || 0 %>
+              </p>
+            </div>
+</div>
+<ul class="post-comments-container">
+          <% if post.recent_comments.any? %>
+            <% post.recent_comments.each do |comment| %>
+              <li>
+                <p><%= User.find_by(id: comment.author_id).name %>: <%= comment.Text[0..60] %>...</p>
+              </li>
+            <% end %>
+          <% else %>
+            <li>No comments added yet.</li>
+          <% end %>
+        </ul>
+<%end%>
+ </article>
+</main>
+ 
+
+
+<%# <%= @user.name%> 
+
+<%# <% @posts.each do |post|%>
+<%# <%=post.title%>
+<%# <% end %> 
+
+<%# <%= @posts.title%> 
+<%# <% @posts.each do |post| %>
+<%# <%= post.text %>
+<%# <% end %> 
+
+<%# <% @comments.each do |comment| %>
+<%# <%= comment.Text%>
+<%# <% end %>
```

Comment:
- [ ] It would be nice if you don't leave the commented piece of code because it makes your code messy and unprofessional 😨 Kindly remove all the commented code from your codebase to make your project follow the Ruby best practices 👍

Reply:
Commented piece of code has been removed from the above mentioned file. Screen shot attached below for your kind reference. 
![image](https://github.com/baqar-abbas/blog-app-rails/assets/117091923/f764b403-9663-4567-9069-c48b8fb355f2)